buildpacks/jvm-function-invoker/src/main.rs

```diff
@@ -0,0 +1,30 @@
+use std::fmt::Debug;
+
+use crate::error::handle_buildpack_error;
+use libcnb::cnb_runtime;
+use libherokubuildpack::HerokuBuildpackErrorHandler;
+use serde::Deserialize;
+
+mod build;
+mod detect;
+mod error;
+mod layers;
+
+fn main() {
+    cnb_runtime(
+        detect::detect,
+        build::build,
+        HerokuBuildpackErrorHandler::new(Box::new(handle_buildpack_error)),
+    );
+}
+
+#[derive(Deserialize, Debug)]
+pub struct JvmFunctionInvokerBuildpackMetadata {
+    runtime: JvmFunctionInvokerBuildpackRuntimeMetadata,
+}
+
+#[derive(Deserialize, Debug)]
+pub struct JvmFunctionInvokerBuildpackRuntimeMetadata {
+    url: String,
+    sha256: String,
+}
```

Comment:
Is `main` the best place for these? I don't know what the best practices are in terms of how minimal it's best to keep main.

Reply:
I think they are in main since they are the main types for this buildpack. It's a judgement call for sure. I could see them being defined in a `buildpack` module where buildpack wide types live.